Sources/App/Core/Search.swift

```diff
@@ -181,28 +183,31 @@ enum Search {
         guard let db = database as? SQLDatabase else {
             fatalError("Database must be an SQLDatabase ('as? SQLDatabase' must succeed)")
         }
-        let mergedTerms = SQLBind(terms.joined(separator: " ").lowercased())
-        // FIXME: bookend with `%`
+        let mergedTerms = terms.joined(separator: " ").lowercased()
+        let searchPattern = "%" + mergedTerms + "%"
 
         return db
             .select()
+            .distinct()
             .column(.keyword)
             .column(keyword)
-            .column(null, as: packageId)
+            .column(nullUUID, as: packageId)
             .column(null, as: packageName)
             .column(null, as: repoName)
             .column(null, as: repoOwner)
-            .column(null, as: score)
+            .column(nullInt, as: score)
             .column(null, as: summary)
-            .column(null, as: stars)
+            .column(nullInt, as: stars)
             .column(null, as: license)
             .column(nullTimestamp, as: lastCommitDate)
             .column(nullTimestamp, as: lastActivityAt)
+            .column(SQLFunction("LEVENSHTEIN", args: keyword, SQLBind(mergedTerms)),
+                    as: levenshteinDist)
             .from(searchView)
             .from(SQLFunction("UNNEST", args: keywords), as: keyword)
-            .where(keyword, .equal, mergedTerms)
-            .limit(1)
-        // TODO: increase limit when we do % matching
+            .where(keyword, ilike, SQLBind(searchPattern))
+            .orderBy(levenshteinDist)
+            .limit(10)
```

Comment:
I picked 10 because it seemed like a sensible figure but I didn't check what the distribution of keyword counts actually looks like. I'd like to have _something_ here but it doesn't have to be 10.

Reply:
In my testing (just using SQL) yesterday, 15 felt good but we can easily change it once it's live.

We may even want to implement something like we have on the package page where they all come back and are hidden by some javascript so they can be expanded if people want to see everything.